### DISCUSSION TITLE
Plotting Return Values for comparison between GPD and GEVD Models

### DISCUSSION THREAD

**Discussion by FranciscoSimoes22** [georgebv/pyextremes#82] [General]
Dear George

I need to plot the Return_Values from GPD model and GEVD Model on the same graph for a visual comparison of both methods.

For instance:

# Plotting Return_values
model_BM.plot_return_values(
    return_period=np.logspace(0.1, 2, 100),
    return_period_size="365.2425D",
    alpha=0.95,
)
plt.show()

and

# Plotting_Return_values
model_POT.plot_return_values(
    return_period = np.logspace(0.1, 2, 100),
    return_period_size="365.2425D",
    alpha=0.95,
)
plt.show()

My question is: I don t need to plot the confidence bounds, i just need both the return_periods "line" on the same graph. Is this possible?

Thank you

**Comment by georgebv** [comment]
You can plot them onto the same axes, you should use the `ax` parameter of the method. Same for confidence - it's controlled by `alpha`.

https://github.com/georgebv/pyextremes/blob/ce530c4925f9c2c42991d03057c965489f21e935/src/pyextremes/eva.py#L1332-L1398
